client/me/developer/dev-survey-notice.tsx

```diff
@@ -0,0 +1,74 @@
+import { Gridicon } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import surveyImage from 'calypso/assets/images/illustrations/developer-survey.svg';
+
+type DevSurveyNotice = {
+	onClose: ( remindTimeInSeconds: number ) => void;
+	onOk: () => void;
+	localeSlug: string;
+};
+
+const ONE_DAY_IN_SECONDS = 24 * 60 * 60;
+const ONE_YEAR_IN_SECONDS = 365 * ONE_DAY_IN_SECONDS;
+
+export const DevSurveyNotice = ( { onClose, onOk, localeSlug }: DevSurveyNotice ) => {
+	const translate = useTranslate();
+
+	const href =
+		localeSlug === 'es'
+			? 'https://wordpressdotcom.survey.fm/developer-survey-es'
+			: 'https://wordpressdotcom.survey.fm/developer-survey';
+
+	return (
+		<div className="developer-survey-notice">
+			<Button
+				className="developer-survey-notice__backdrop"
+				onClick={ () => onClose( ONE_YEAR_IN_SECONDS ) }
+			/>
+			<div className="developer-survey-notice__popup">
+				<div className="developer-survey-notice__popup-head">
+					<div className="developer-survey-notice__popup-head-title">
+						{ translate( 'Developer Survey' ) }
+					</div>
+					<Button
+						onClick={ () => onClose( ONE_YEAR_IN_SECONDS ) }
+						className="developer-survey-notice__popup-head-close"
+					>
+						<Gridicon icon="cross" size={ 16 } />
+					</Button>
+				</div>
+				<div className="developer-survey-notice__popup-img">
+					<img src={ surveyImage } alt="" />
```

Comment:
Can we set a default height to the image container so the height of the prompt remains the same before and after the image is loaded? That way, we avoid getting something like this:

https://github.com/Automattic/wp-calypso/assets/104910361/1eed1662-9cc7-43f0-8a9f-454f34b97263

Reply:
We should also provide something for the `alt` attribute (and that is localized), e.g.:
```suggestion
					<img src={ surveyImage } alt={ translate( 'Code editor' ) } />
```